subtrack.DAL/Entities/DateTimeSetting.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace subtrack.DAL.Entities
+{
+    public class DateTimeSetting : SettingsBase
+    {
+        public const string Key = "LastAutoPaymentTimeStamp";
```

Comment:
My bad, but we should probably name this LastAutoPaymentTimeStampKey since DateTimeSetting can be used for any setting